core/src/main/java/org/elasticsearch/common/logging/LogConfigurator.java

```diff
@@ -58,12 +58,14 @@
      * Configure logging without reading a log4j2.properties file, effectively configuring the
      * status logger and all loggers to the console.
      *
-     * @param settings for configuring logger.level and individual loggers
+     * @param loggerLevel for configuring logger.level
      */
-    public static void configureWithoutConfig(final Settings settings) {
-        Objects.requireNonNull(settings);
+    public static void configureWithoutConfig(final String loggerLevel) {
```

Comment:
This situation feels very fragile to me. It means in the future if we do want a settings instance here because we want to control fine-grained logging settings for a CLI tool, we can not without accidentally reintroducing the problem.